server/src/test/java/org/spine3/server/command/CommandBusShould.java

```diff
@@ -83,26 +89,31 @@
     private TestResponseObserver responseObserver;
 
     @Before
-    public void setUp() {
+    public void setUpTest() {
```

Comment:
And if you suggest the renaming here, are we going to rename all other similar methods in other framework tests?